Src/library/base/elks/support/reflected_copy_semantics_object.e

```diff
@@ -118,20 +118,16 @@ feature {NONE} -- Implementation
 
 	dereference (ptr: POINTER; offset: INTEGER): POINTER
 		external
-			"C inline use %"eif_built_in.h%""
-		alias
-			"return *(EIF_REFERENCE *) eif_obj_at($ptr, $offset);"
+			"built_in static"
```

Comment:
I thought it was eventually possible to reuse the {ISE_RUNTIME}.raw_reference_field_at_offset ?

Reply:
I will check

Reply:
ok here we can do
```dereference (ptr: POINTER; offset: INTEGER): POINTER
		do
			Result :=  {ISE_RUNTIME}.raw_reference_field_at_offset (ptr, offset)
		end
```

Reply:
Done

Reply:
I think the more we have in the Eiffel classes, the better it is.
built-in facility is really useful for portability, and for compatibility with Gobo,
but when using it .. it is not nice to see what is really implemented.

Reply:
maybe we could improve EiffelStudio to show the real implementation.